src/plugins/data/common/index_patterns/index_patterns/index_patterns.ts

```diff
@@ -29,7 +29,10 @@ import {
   FieldSpec,
   IndexPatternFieldMap,
 } from '../types';
-import { FieldFormatsStartCommon } from '../../field_formats';
+import {
+  FieldFormatsStartCommon,
+  UI_SETTINGS as FORMATS_UI_SETTINGS,
```

Comment:
Nit - I'd rather see a globally unique name on this, `UI_SETTINGS` might get confused with the UiSettings service

Reply:
Like export this from a plugin, let's say, as `FORMATS_UI_SETTINGS` instead of `UI_SETTINGS` ?